text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
```

Comment:
Since this isn't in the RFC repo, I think we should remove this, because these fields are mostly specific to the RFC repo. And maybe also drop the `0006-` from the file name (I'm not sure if we have an RFC 0006 or not but it would certainly conflict with that).